### ISSUE TITLE
Empty groups with zero users, no messages

### ISSUE THREAD

**Issue by gammatrapdoor** [EionRobb/purple-discord#442]
Hello. I am running this software on Windows XP and the current nightly version seems to not be fetching any of the channels actively, they populate the buddy list but are all offline/empty with zero people inside. Neither does it show any individual DMs, and the channel list does not reflect leaving servers at all. Additionally I got an error when idling that said "websocket failed to send". It seems there are some other issues with this occuring before, but the issues themselves seemed old and I wanted to push a new one. If anyone has any fixes for this issue let me know.

**Comment by iv4nhoe** [comment]
Is this with an existing account or have you tried to remove the account from Pidgin and re-add it?
Are there any errors in the Debug Window (Help > Debug Window before connecting the account) that could offer more insight of what is going wrong?

**Comment by gammatrapdoor** [comment]
Hello, I've tried removing and readding my account multiple times since doing this. I must add, I have had Pidgin open just to see if anything happens, and I still get an occasional chat popping up with repeated "This account has disconnected and you are no longer in this chat" messages. I tried running debug when I re-added my account and it had a lot to spit out, but I noticed it contains a lot of the information of who is on my friends list/what server I'm in so I'm hesitant to share this publicly. 


As of writing this it magically seemed to start working, I will keep you in the loop if I run into any other issues with it! Thank you for your help regardless :)

**Comment by gammatrapdoor** [comment]
Hello, I've tried removing and readding my account multiple times since doing this. I must add, I have had Pidgin open just to see if anything happens, and I still get an occasional chat popping up with repeated "This account has disconnected and you are no longer in this chat" messages. I tried running debug when I re-added my account and it had a lot to spit out, but I noticed it contains a lot of the information of who is on my friends list/what server I'm in so I'm hesitant to share this publicly.

As of writing this it magically seemed to start working, I will keep you in the loop if I run into any other issues with it! Thank you for your help regardless :)
